pkg/apis/openebs.io/v1alpha1/cas_keys.go

```diff
@@ -54,6 +54,18 @@ const (
 	// PersistentVolumeClaimKey is the key to fetch name of PersistentVolumeClaim
 	PersistentVolumeClaimKey CASKey = "openebs.io/persistentvolumeclaim"
 
+	// NodeAffinityReplicaJivaIsPatchHeaderKey is the key to fetch value of IsPatchKey
+	// its value is "Enabled".
+	NodeAffinityReplicaJivaIsPatchHeaderKey CASKey = "Node-Affinity-Replica-Jiva-Is-Patch"
+
+	// NodeAffinityReplicaJivaIsPatchKey is the key to fetch value of IsPatchKey
+	// its value is "Enabled".
+	NodeAffinityReplicaJivaIsPatchKey CASKey = "nodeAffinity.replica.jiva.openebs.io/is-patch"
```

Comment:
please remove this now. This naming convention applies to dot convention only.

Reply:
Using this as part of annotation, so it can't be removed. This will be used while instantiating engine.